VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -531,37 +523,22 @@ protected void hideSoftKeypad() {
     }
 
     public boolean showErrorMessage(String header, String message) {
-        return showErrorMessage(header, message, true);
-    }
-
-    public boolean showErrorMessage(String header, String message, boolean isPersistent) {
-        LinearLayout error_layout = (LinearLayout) findViewById(R.id.error_layout);
-        if (error_layout == null) {
-            logger.warn("Error Layout not available, so couldn't show flying message");
-            return false;
-        }
-        TextView errorHeader = (TextView) findViewById(R.id.error_header);
-        TextView errorMessageView = (TextView) findViewById(R.id.error_message);
-        if (header == null || header.isEmpty()) {
-            errorHeader.setVisibility(View.GONE);
-        } else {
-            errorHeader.setVisibility(View.VISIBLE);
-            errorHeader.setText(header);
-        }
-        if (message != null) {
-            errorMessageView.setText(message);
-        }
-        ViewAnimationUtil.showMessageBar(error_layout, isPersistent);
+        hideErrorMessage();
+        alertDialog = new AlertDialog.Builder(this)
```

Comment:
Once this change has been implemented, `showErrorMessage` and `showErrorDialog` methods would essentially be doing the same thing, so, it would be better to call `showErrorDialog` method and remove `showErrorMessage`.